chefspec.gemspec

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |s|
   s.files         = %w{LICENSE Rakefile Gemfile chefspec.gemspec} + Dir.glob("{lib,templates,spec}/**/*", File::FNM_DOTMATCH).reject { |f| File.directory?(f) }
   s.require_paths = ["lib"]
 
-  s.required_ruby_version = ">= 2.5"
+  s.required_ruby_version = ">= 3.3"
```

Comment:
We still have Chef 17 on 3.0 (in active dev) and Chef 18 on 3.1 (in active dev). Thus far, we don't have any Chef Client pipelines on 3.3